device/relayagent/v4/dhcpv4relayagent.yaml

```diff
@@ -0,0 +1,37 @@
+components:
+  schemas:
+    V4RA.Inst:
```

Comment:
The object names can be set as Ipv4RelayAgent.Instance. 
I think these names do get exposed via gosnappi in case user wants to modify something in a config built by another function for example.

Reply:
ok but we are not considering to merge relay agent to master, this is juyts rough sketch